lib/v2/components/search_user/components/search_result_list.dart

```diff
@@ -1,34 +1,38 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:seeds/v2/components/search_user/components/search_result_row.dart';
+import 'package:seeds/v2/components/search_user/interactor/search_user_bloc.dart';
+import 'package:seeds/v2/components/search_user/interactor/viewmodels/search_user_state.dart';
 import 'package:seeds/v2/datasource/remote/model/member_model.dart';
 
 class SearchUsersList extends StatelessWidget {
-  final List<MemberModel> users;
   final ValueSetter<MemberModel> resultCallBack;
 
-  const SearchUsersList({Key? key, required this.users, required this.resultCallBack}) : super(key: key);
+  const SearchUsersList({Key? key, required this.resultCallBack}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    return Expanded(
-      child: ListView.builder(
-          itemCount: users.length,
-          itemBuilder: (BuildContext context, int index) {
-            MemberModel user = users[index];
-            return GestureDetector(
-              onTap: () {
-                resultCallBack(user);
-              },
-              child: Padding(
-                padding: const EdgeInsets.only(top: 16),
-                child: SearchResultRow(
-                  account: user.account,
-                  name: user.nickname,
-                  imageUrl: user.image,
+    return BlocBuilder<SearchUserBloc, SearchUserState>(builder: (context, SearchUserState state) {
+      return Expanded(
+        child: ListView.builder(
+            itemCount: state.users.length,
+            itemBuilder: (BuildContext context, int index) {
+              MemberModel user = state.users[index];
+              return GestureDetector(
+                onTap: () {
+                  resultCallBack(user);
```

Comment:
I would have done it with an event (OnUserSeleceted) and a listener 
(listener: (_,state)=> resultCallBack(state.userSelected)) under the `BlocProvider <SearchUserBloc>` 
So the callback is not passed in each `case`, but it does its job anyway 👍